docs/README.md

```diff
@@ -202,6 +202,13 @@ Feel free to use the data to construct your own data application and you can ref
       <td><a href="https://github.com/X-lab2017/open-digger/blob/master/src/metrics/chaoss.ts#L494">Link</a></td>
       <td><a href="https://codepen.io/frank-zsy/pen/VwBqwaP?type=change_request_age">Demo</a></td>
     </tr>
+    <tr>
```

Comment:
hi, there may have some format problems, can you have a look?
![Uploading image.png…]()

Reply:
> hi, there may have some format problems, can you have a look? <img alt="image" width="1027" src="https://user-images.githubusercontent.com/37795442/238812279-0049bb12-ee23-4f5c-b8f2-65f722a0ff78.png">

This should be in `docs/zh-cn/README.md`. I marked the wrong position

Reply:
That was a mistake :) fixed it.